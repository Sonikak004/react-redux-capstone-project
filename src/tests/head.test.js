import React from 'react';
import { render } from '@testing-library/react';

import Header from '../components/head';

describe('Header component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
