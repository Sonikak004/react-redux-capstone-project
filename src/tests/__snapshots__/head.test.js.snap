// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <div
      class="header-left"
    >
      <h1>
        Country Explorer
      </h1>
    </div>
    <div
      class="header-right"
    >
      <button
        class="icon-button"
        type="button"
      >
        <i
          class="fas fa-microphone"
        />
      </button>
      <button
        class="icon-button"
        type="button"
      >
        <i
          class="fas fa-cog"
        />
      </button>
    </div>
  </header>
</DocumentFragment>
`;
